package.json

```
@@ -12,7 +12,7 @@
   "author": "sebworks",
   "private": true,
   "scripts": {
-    "dev": "cross-env ENABLE_AUDIO=true nuxt",
+    "dev": "node src/locales/scripts/get-validated-locales && cross-env ENABLE_AUDIO=true nuxt",
```

We could use the `get-translations` script; or write a different one to just get the validated locales and call it?
```suggestion
    "dev": "npm run i18n:get-translations && cross-env ENABLE_AUDIO=true nuxt",
```